01-js-intro-and-jquery/07/js/scripts.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function() {
+	var alias = '';
+	var focusAlias = function() {	
+		$('.alias').focus(); 
+	};
+	var getResponse = function() {
+		alias = $('.alias').val();
+
+		$.ajax({
+   url: "http://bootcamp.aws.af.cm/welcome/" + alias,
```

Comment:
**Minor:** indentation seems weird. Also, per code convention, try to always use single quotes within JS files (double quotes are required for JSON).